1.0.1/jessie/product/README.md

```diff
@@ -17,3 +17,25 @@ This image contains:
 ## Supported tags
 
 - [`1.0.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/product/Dockerfile)
+
+## Example Usage
+
+1. Create a Dockerfile for your application, the following example assumes you have already compiled your application (which is the expected use case for this image)
+
+  ```
+  FROM microsoft/aspnetcore
+  WORKDIR /app
+  ADD /app
+  ENTRYPOINT ["dotnet", "myapp.dll"]
+  ```
+
+2. Build and run your app:
+
+  ```
+  $ docker build -t myapp .
+  $ docker run -d -P myapp
```

Comment:
You might confuse people by using the `publish-all` flag. I'd much rather call out an explicit port e.g., `-p 8000:80`.